platform-sdk/platform-apps/tests/ConsistencyTestingTool/src/main/java/com/swirlds/demo/consistency/ConsistencyTestingToolState.java

```diff
@@ -54,17 +55,38 @@ private static class ClassVersion {
     }
 
     /**
-     * The history of transactions that have been handled by this app
+     * The history of transactions that have been handled by this app.
      * <p>
-     * A deep copy of this object is NOT created when this state is copied
+     * A deep copy of this object is NOT created when this state is copied. This object does not affect the hash of this
+     * node.
      */
     private final TransactionHandlingHistory transactionHandlingHistory;
 
     /**
-     * The true "state" of this app. This long value is updated with every transaction, and with every round
+     * The true "state" of this app. This long value is updated with every transaction, and with every round.
+     * <p>
+     * Effects the hash of this node.
      */
     private long stateLong = 0;
 
+    /**
+     * The number of rounds handled by this app. Is incremented each time
+     * {@link #handleConsensusRound(Round, SwirldDualState)} is called. Note that this may not actually equal the round
+     * number, since we don't call {@link #handleConsensusRound(Round, SwirldDualState)} for rounds with no events.
+     *
+     * <p>
+     * Effects the hash of this node.
+     */
+    private long roundsHandled = 0;
+
+    /**
+     * If not null and we are handling the first round after genesis, configure a freeze this many seconds later.
```

Comment:
Shouldn't it say "if not 0 and we are handling the first round..."?

Reply:
Fixed. Originally I was storing null here, but the configuration engine doesn't support null instants.